src/modules/accessoryPanelConstants.js

```diff
@@ -0,0 +1,12 @@
+export const ACCESSORY_PANEL_TAB_INPUT = 'INPUT'
+export const ACCESSORY_PANEL_TAB_PROGRAM = 'PROGRAM'
+export const ACCESSORY_PANEL_TAB_YOUR = 'YOUR'
+export const ACCESSORY_PANEL_TAB_TRACKED_ENTITY = 'TRACKED_ENTITY'
+export const ACCESSORY_PANEL_DEFAULT_WIDTH = 260
+export const ACCESSORY_PANEL_MIN_WIDTH = 180
+export const ACCESSORY_PANEL_MIN_PX_AT_END = 50
+export const PRIMARY_PANEL_WIDTH = 260
+export const ACCESSORY_PANEL_WIDTH_STORAGE_KEY =
```

Comment:
I wonder if we have some conventions about the naming.
In dashboard we have a couple of keys and the naming there is "prefixed" with`dhis2.dashboard.`.
I don't think it matters really, but we do seem to use different formats.

Reply:
Perhaps it would be good to have a consistent approach across different apps, but I think that would be out-of-scope for this PR. Especially because quite a few of these constants were pre-existing, I added a few but others were only moved from one file to another. The ones that were added used the same naming conventions as the ones already present.